contents/stats.md

```diff
@@ -0,0 +1,445 @@
+sco("correct_types()"; process=without_caption_label)
+```
+
+We can compute the mode with the `combine` function from `DataFrames.jl`.
+Notice that, unlike previously with mean and median, we will not group our data:
+
+```jl
+# mode # hide
+s = """
+    combine(correct_types(), :age => mode)
+    """
+sco(s; process=without_caption_label)
+```
+
+### Visualization of Central Tendencies {#sec:stats_central_vis}
+
+In order to have a better intuition behind the difference between mean, median and mode, it is always useful to make some visualizations.
+We will cover statistical visualizations in depth in @sec:stats_vis.
+Below, in @fig:plot_central, we have two data distributions:
+
+- **Upper row**: **normal** distributed data
+- **Lower row**: **non-normal** distributed data
+
+```jl
+s = """
+    fig = plot_central()
+    caption = "Normal and Non-Normal Distributed Data -- Differences Between Central Tendencies."
+    label = "plot_central"
+    Options(fig; filename=label, caption, label)
+    """
+sco(s)
+```
+
+You can see that the mean, median and mode are almost the same in the normal distributed data, but they differ a lot in the non-normal distributed data.
+In the non-normal distributed data the mean is highly skewed towards to the right, *biasing* our central tendency.
+This is an example of an outlier scenario where the mean can be highly sensitive to influential observations.
+Nevertheless, the median is unaffected by the outliers and can be used as a *reliable* central tendency.
+This demonstrates that the **median is robust to outliers**.
+In both cases, the mode is used only for comparison, since it is not advised for use with numerical data.
+
+### Advice on Central Tendencies {#sec:stats_central_advice}
+
+You might be wondering: "which central tendency shall I use? Mean? Median? Mode?".
+Here is our advice:
+
+- For data that do *not* have outliers, use the **mean**
+- For data that *do* have outliers, use the **median**
+- For categorical/nominal data, use the **mode**
+
+## Dispersion Measures {#sec:stats_dispersion}
+
+In statistics, **dispersion is a measure of how much spread certain observations are from a central tendency**.
+It is also called variability or spread.
+One interesting property is that, contrary to central tendencies, dispersion measures can *only* be **positive**.
+In other words, we do not have negative measures of dispersion.
+
+### Variance and Standard Deviation {#sec:stats_dispersion_varstd}
+
+The first dispersion measure that we will cover are variance and standard deviation.
+We will be covering both measures together because they have a profound relationship.
+**The variance is the square of the standard deviation and the standard deviation is the square root of the variance.**
+
+Let's start with the variance. **Variance is mean of the squared difference between measurements and their average value**:
+
+$$ \operatorname{Var}(x) = \frac{1}{n-1} \sum^n_{i=1} (x_i - \bar{x})^2. $$ {#eq:variance}
```

Comment:
In the equations, `median` was in all lowercase, `Var` and `Cov` start with an uppercase and `MAD` is in all uppercase. Maybe, this can be made more consistent? I think that `Var` and `var` are both used in practice in equations depending on the domain.

Reply:
Let's wait for @lazarusA opinions on math stuff.

Reply:
I say, let's do lower case. Less confusion later when calling the function `cov` or `var`.

Reply:
Ok